controllers/users.js

```diff
@@ -196,8 +196,9 @@ const postUserPicture = async (req, res) => {
  */
 const updateUser = async (req, res) => {
   try {
-    const userId = req.params.userId;
-    const { id: profileDiffId, message, ...profileDiff } = req.body;
+    const { id: profileDiffId, message } = req.body;
+
+    const { approval, timestamp, userId, ...profileDiff } = await profileDiffsQuery.fetchProfileDiff(profileDiffId);
```

Comment:
I believe we should handle error here when we get we get empty data and remove the error handler at line 211
```js
if (profileResponse.notFound) return res.boom.notFound("Profile Diff doesn't exist");
```